lib/cocoapods/bazel.rb

```diff
@@ -135,18 +135,15 @@ def self.format_files(build_files:, buildifier:, workspace:)
       when true
         return unless Pod::Executable.which('buildifier')
 
-        args = ['buildifier']
+        args = %w[buildifier -type build] + build_files.each_key.map { |d| File.join workspace, d, 'BUILD.bazel' }
       when String, Array
         args = Array(buildifier)
```

Comment:
This line originally accepted an arbitrary array for  `buildifer`: e.g. today, can't you pass `bazel run :buildifier` Mainly it sets the `executable` to that below

Reply:
Undid this, totally right!